test/dependent_delays.jl

```diff
@@ -0,0 +1,34 @@
+using DelayDiffEq, DiffEqProblemLibrary, Base.Test
+
+alg = MethodOfSteps(BS3())
+
+dde_int = init(prob_dde_1delay, alg)
+
+sol = solve!(dde_int)
+
+@test sol.errors[:l∞] < 3.7e-5
+@test sol.errors[:final] < 2.0e-5
+@test sol.errors[:l2] < 1.5e-5
+
+# constant delay specified as function
```

Comment:
We should test this at lower tolerances too.

Reply:
I'm not completely sure what's missing... Just the same tests with lower tolerances, again with constant delays, dependent delays, and without delays?

Reply:
Just dependent delays with the solver at lower tolerance, show that the errors get lower.

Reply:
OK, I added some additional tests at lower tolerances. Is this what you wanted to see?

Reply:
Yeah looks good.